### ISSUE TITLE
[FR] print mappingproxy like a dict

### ISSUE THREAD

**Issue by Cologler** [tommikaikkonen/prettyprinter#9]
If the idea was accepted, I can create a PR.

**Comment by tommikaikkonen** [comment]
Sounds great! We can follow how Python renders them, like

```python
mappingproxy({
    'a': 1,
    'b': 2,
})
```

or alternatively we could display the MappingProxyType constructor, which is used to create mappingproxies:

```python
types.MappingProxyType({
    'a': 1,
    'b': 2,
})
```

The former is more succinct, but the later is how you would build an actual mapping proxy value. Not sure which one is best, you can pick the one you prefer and we can change it in a later release if it becomes an issue.

**Comment by Cologler** [comment]
close by #11

**Comment by tommikaikkonen** [comment]
This fix is included in 0.9.0 that I just uploaded to PyPi. Thanks!
